tests/server.py

```diff
@@ -474,6 +474,18 @@ def getHostByName(
                     return fail(DNSLookupError("OH NO: unknown %s" % (name,)))
                 return succeed(lookups[name])
 
+        # In order for the TLS protocols to use the proper we need to patch
```

Comment:
"the proper"? The proper reactor's clock?

Reply:
```suggestion
        # In order for the TLS protocols to use the proper reactor's clock we need to patch
```